pkg/kubelet/prober/worker.go

```diff
@@ -230,7 +244,7 @@ func (w *worker) doProbe() (keepGoing bool) {
 	if c.Started != nil && *c.Started {
 		// Stop probing for startup once container has started.
 		if w.probeType == startup {
-			return true
+			return false
```

Comment:
`startupProbe` behaviour changed
After the container restarts, `startupProbe` will no longer be executed
#101093